llvm/lib/Target/NVPTX/NVPTXSubtarget.h

```diff
@@ -89,6 +89,9 @@ class NVPTXSubtarget : public NVPTXGenSubtargetInfo {
     return SmVersion >= 100 && PTXVersion >= 88 &&
            AS == NVPTXAS::ADDRESS_SPACE_GLOBAL;
   }
+  bool hasUsedBytesMaskPragma() const {
+    return SmVersion >= 50 && PTXVersion >= 83;
+  }
```

Comment:
Do we still need it?

Reply:
Yes, this gets used in ISELLowering now.